Sources/FigmaExport/Loaders/ImagesLoader.swift

```diff
@@ -26,26 +26,106 @@ final class ImagesLoader {
         self.logger = logger
     }
 
-    func loadIcons(filter: String? = nil) throws -> [ImagePack] {
+    func loadIcons(filter: String? = nil) throws -> (light: [ImagePack], dark: [ImagePack]?) {
+        if params.common?.icons?.useSingleFile == true {
```

Comment:
Existing FigmaExport users doesn't have `useSingleFile` property in the `figma-export.yml` file. This will cause a bug. FigmaExport will terminate with the following error because dark icons wont found.
`Error: Components not found in the Figma file.`

Reply:
Done ✅

Reply:
Run `figma-export icons` for example project `./Examples/Example/`. The bug is still here. FigmaExport tries to download dark icons but there is no dark icons in the example Figma file and it terminates with the `Components not found` error.

To fix this issue update code of the `loadIconsFromLightAndDarkFile` method by ignoring any errors:
```swift
let darkIcons = params.figma.darkFileId.flatMap { fileId in
    try? _loadImages(
        fileId: fileId,
        frameName: iconsFrameName,
        params: formatParams,
        filter: filter)
}
```

Reply:
Thanks! I fixed this, including for darkImages